src/internal/modules/account/controller.js

```diff
@@ -95,7 +101,7 @@ const getDeleteUserAccount = async (request, h, formFromPost) => {
     ...request.view,
     userEmail,
     form,
-    back: `/user/${userId}/status`
+    back: `/accounts`
```

Comment:
You can get to this page from 2 different places, this might need to be more flexible to reflect this